### PR TITLE
Returning value to supress log errors

### PR DESCRIPTION
**Pull request by weotch**
When nothing is returned, the functions log contains:

> Function returned undefined, expected Promise or value

### CODE REVIEW

**Comment by weotch** [comment]
I signed it!

File: generate-thumbnail/functions/index.js [hunk 1]
```diff
@@ -51,19 +51,19 @@ exports.generateThumbnail = functions.storage.object().onChange(event => {
   // Exit if this is triggered on a file that is not an image.
   if (!event.data.contentType.startsWith('image/')) {
     console.log('This is not an image.');
-    return;
+    return 0;
```
Comment (Brianvdb) [inline generate-thumbnail/functions/index.js]: null

File: generate-thumbnail/functions/index.js [hunk 2]
```diff
@@ -51,19 +51,19 @@ exports.generateThumbnail = functions.storage.object().onChange(event => {
...
   }
 
   // Exit if the image is already a thumbnail.
   if (fileName.startsWith(THUMB_PREFIX)) {
     console.log('Already a Thumbnail.');
-    return;
+    return 0;
```
Comment (Brianvdb) [inline generate-thumbnail/functions/index.js]: null

File: generate-thumbnail/functions/index.js [hunk 3]
```diff
@@ -51,19 +51,19 @@ exports.generateThumbnail = functions.storage.object().onChange(event => {
...
   }
 
   // Exit if this is a move or deletion event.
   if (event.data.resourceState === 'not_exists') {
     console.log('This is a deletion event.');
-    return;
+    return 0;
```
Comment (Brianvdb) [inline generate-thumbnail/functions/index.js]: null

**Comment by nicolasgarnier** [comment]
Thanks @weotch :)

**Pull request merged**